src/engine/input/ControlWithInput.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Godot;
+
+/// <summary>
+///   Input class for a <see cref="Godot.Control"/>
```

Comment:
This documentation should be more comprehensive. This should say what being an "input class" means.